src/ray/rpc/rpc_chaos.cc

```diff
@@ -76,35 +76,35 @@ class RpcFailureManager {
       // 25% chance
       num_remaining_failures--;
       return RpcFailure::Request;
-    } else if (rand == 1) {
+    }
+    if (rand == 1) {
       // 25% chance
       num_remaining_failures--;
       return RpcFailure::Response;
-    } else {
-      // 50% chance
-      return RpcFailure::None;
     }
+    // 50% chance
+    return RpcFailure::None;
   }
 
  private:
   absl::Mutex mu_;
   std::mt19937 gen_;
   // call name -> # remaining failures
-  std::unordered_map<std::string, uint64_t> failable_methods_ ABSL_GUARDED_BY(&mu_);
+  absl::flat_hash_map<std::string, uint64_t> failable_methods_ ABSL_GUARDED_BY(&mu_);
```

Comment:
Apart from the performance improvement, another benefit is modern interfaces (C++20/23 features).